### ISSUE TITLE
在 SecretFlow 中提供自定义 loss function 的教程

### ISSUE THREAD

**Issue by Candicepan** [secretflow/secretflow#1044]
> 此 ISSUE 为 [隐语开源共建计划（SecretFlow Open Source Contribution Plan，简称 SF OSCP）第三期](https://github.com/orgs/secretflow/discussions/1007)任务 ISSUE，欢迎社区开发者参与共建～
> 若有感兴趣想要认领的任务，但还未报名，辛苦先完成[报名](https://www.wjx.top/vm/QKio0dq.aspx#)进行哈～

## 任务介绍
- **任务名称**：在 SecretFlow 中提供自定义 loss function 教程
- **技术方向**：FL/application
- **任务难度**：热身🌟

## 详细要求
- 安全性：尽量降低数据、模型泄露的可能
- 功能性：在 SecretFlow 中开发自定义 loss function 的文档，介绍如何定义复杂的 loss function。
- 有效性：需要跑出实验验证教程中的自定义loss function的正确性。
- 代码规范：Python 代码需要使用 black+isort 进行格式化（流水线包含代码规范检查卡点）
- 框架后端：尽量同时覆盖 TensorFlow 和 Pytorch 的自定义 loss fn。
- 提交说明：关联该 isuue 并提交代码至 
https://github.com/secretflow/secretflow/tree/main/docs/tutorial

## 能力要求
- 了解基本 git 操作
- 熟悉 深度学习、联邦学习
- 对 SecretFlow 有一定了解

## 操作说明
- 多语言操作说明：https://github.com/secretflow/secretflow/blob/main/CONTRIBUTING.md

**Comment by Mingbo-Lee** [comment]
Mingbo-Lee Give it to me

**Comment by reacher1130** [comment]
reacher1130 Give it to me

**Comment by Candicepan** [comment]
> reacher1130 Give it to me

hello～该任务即将进入 PR 提交阶段呢～感谢您的认领，您可以看一下其他类似的任务进行认领共建呀～https://github.com/orgs/secretflow/projects/6

**Comment by Candicepan** [comment]
> reacher1130 Give it to me

欢迎添加隐语小助手微信：SecretFlow04 进一步沟通交流呀～
